light-system-programs/programs/merkle_tree_program/src/message_merkle_tree/initialize.rs

```diff
@@ -16,7 +16,7 @@ pub struct InitializeNewMessageMerkleTree<'info> {
     /// CHECK: it should be unpacked internally
     #[account(
         init,
-        seeds = [&program_id.to_bytes()[..], MESSSAGE_MERKLE_TREE_SEED],
+        seeds = [&__program_id.to_bytes()[..], MESSSAGE_MERKLE_TREE_SEED],
```

Comment:
Because Anchor did so. We are not defining that variable ourselves, it's being generated inside `#[program]` attribute macro.

The change was made in this commit:

https://github.com/coral-xyz/anchor/commit/2e89b79c51d6a4e2d14be1bd5de7e6387e642efc

Reply:
I see thanks